sdcm/tester.py

```diff
@@ -804,10 +804,10 @@ def is_encrypt_keys_needed(self):
 
     def prepare_kms_host(self) -> None:
         if (self.params.is_enterprise and ComparableScyllaVersion(self.params.scylla_version) >= '2023.2.0~rc0'
-                    and self.params.get('cluster_backend') == 'aws'
-                    and not self.params.get('scylla_encryption_options')
-                    and self.params.get("db_type") != "mixed_scylla"  # oracle probably doesn't support KMS
-                ):
+            and self.params.get('cluster_backend') == 'aws'
+            and not self.params.get('scylla_encryption_options')
+            and self.params.get("db_type") != "mixed_scylla"  # oracle probably doesn't support KMS
+            ):
```

Comment:
This part got changed without any reason, again.
Please, exclude such stuff when you do `git commit` by adding the `-p` flag which allows to choose changes you want to add to a commit.

Reply:
when it deems necessary I do, in this case it's autopep8 doing, I guess it's a bug that I'm not going to chase now. (or ever)

Reply:
I am not talking about fixing bugs.
I am talking about removing garbage from the commit.
Those lines should have stayed unchanged.

Reply:
not important enough